package.json

```diff
@@ -5,12 +5,9 @@
     "@rails/actioncable": "^6.0.0-alpha",
     "@rails/activestorage": "^6.0.0-alpha",
     "@rails/ujs": "^6.0.0-alpha",
-    "@rails/webpacker": "^5.4.3",
     "turbolinks": "^5.2.0"
   },
   "license": "AGPL-3.0-or-later",
   "version": "0.1.0",
-  "devDependencies": {
-    "webpack-dev-server": "^4.9.0"
-  }
+  "devDependencies": {}
 }
```

Comment:
We may be able to delete this entire file, since I don't thik we use UJS, ActiveStorage, ActionCable or TurboLinks in this project...